tests/Faithlife.Build.Tests/BuildRunnerTests.cs

```diff
@@ -15,4 +15,157 @@ public void NullInitializeThrows()
 	{
 		Assert.Throws<ArgumentNullException>(() => BuildRunner.Execute([], null!));
 	}
+
+	[Test]
+	public void FailsOnMissingTarget()
+	{
+		using var error = new StringWriter();
+		Console.SetError(error);
```

Comment:
I don't love the non-parallelizable tests, but I guess we don't have a choice.

I'm surprised they aren't being run in parallel, but maybe that's not the default.

Reply:
Another option would be to pass stdout/stderr streams via `BuildRunnerSettings` and have it prefer those over `Console` if they're present.

Reply:
Accord to this, there is no parallel execution in NUnit by default: https://docs.nunit.org/articles/nunit/writing-tests/attributes/parallelizable.html

Reply:
That could work for direct Faithlife.Build output, but not for the apps that targets run, which could be confusing.

Reply:
Seeing as parallel execution is not the default in NUnit, I am going to say the `Console` redirection is fine as is.